### PR TITLE
feat: add log query interface (#362)

### LINKED ISSUE #362: Add Log Query Service
## Proposal
add log interfaces for the following integration

### PR DESCRIPTION
**Pull request by zzhb101**
# Which issue does this PR close?

Closes #362

# Rationale for this change
 

# What changes are included in this PR?

# Are there any user-facing changes?

# How does this change test

### CODE REVIEW

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/Log.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+package io.holoinsight.server.home.biz.service.log;
+
+import lombok.Data;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author zzhb101
+ * @time 2023-04-03 7:06 下午
```
Comment (masaimu) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/Log.java:13]: Comments should be written in English.

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogContextRequest.java [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+ */
+package io.holoinsight.server.home.biz.service.log;
+
+import lombok.Data;
+
+import java.util.Map;
+
+/**
+ * @author zzhb101
+ * @time 2023-04-03 6:13 下午
```
Comment (masaimu) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogContextRequest.java:12]: Comments should be written in English.

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogContextResponse.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+package io.holoinsight.server.home.biz.service.log;
+
+import lombok.Data;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author zzhb101
+ * @time 2023-04-03 6:17 下午
```
Comment (masaimu) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogContextResponse.java:13]: Comments should be written in English.

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogCountRequest.java [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * @author zzhb101
+ * @time 2023-04-03 5:28 下午
```
Comment (masaimu) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogCountRequest.java:14]: Comments should be written in English.

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogCountResponse.java [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+ */
+package io.holoinsight.server.home.biz.service.log;
+
+import lombok.Data;
+
+import java.util.Map;
+
+/**
+ * @author zzhb101
+ * @time 2023-04-03 5:59 下午
```
Comment (masaimu) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogCountResponse.java:12]: Comments should be written in English.

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogKey.java [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+ */
+package io.holoinsight.server.home.biz.service.log;
+
+import lombok.Data;
+
+import java.util.Map;
+
+/**
+ * @author zzhb101
+ * @time 2023-04-13 1:55 下午
```
Comment (masaimu) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogKey.java:12]: Comments should be written in English.

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogMultiQueryRequest.java [hunk 7]
```diff
@@ -0,0 +1,72 @@
...
+package io.holoinsight.server.home.biz.service.log;
+
+import lombok.Data;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author zzhb101
+ * @time 2023-04-03 8:31 下午
```
Comment (masaimu) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogMultiQueryRequest.java:13]: Comments should be written in English.

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogMultiQueryResponse.java [hunk 8]
```diff
@@ -0,0 +1,48 @@
...
+package io.holoinsight.server.home.biz.service.log;
+
+import lombok.Data;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author zzhb101
+ * @time 2023-04-03 8:49 下午
```
Comment (masaimu) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogMultiQueryResponse.java:13]: Comments should be written in English.

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogPathRequest.java [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+ */
+package io.holoinsight.server.home.biz.service.log;
+
+import lombok.Data;
+
+import java.util.Map;
+
+/**
+ * @author zzhb101
+ * @time 2023-04-03 5:25 下午
```
Comment (masaimu) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogPathRequest.java:12]: Comments should be written in English.

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogQueryRequest.java [hunk 10]
```diff
@@ -0,0 +1,68 @@
...
+package io.holoinsight.server.home.biz.service.log;
+
+import lombok.Data;
+
+import java.util.ArrayList;
+import java.util.Map;
+
+/**
+ * @author zzhb101
+ * @time 2023-04-03 5:28 下午
```
Comment (masaimu) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogQueryRequest.java:13]: Comments should be written in English.

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogQueryResponse.java [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+ */
+package io.holoinsight.server.home.biz.service.log;
+
+import lombok.Data;
+
+import java.util.List;
+
+/**
+ * @author zzhb101
+ * @time 2023-04-03 8:05 下午
```
Comment (masaimu) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogQueryResponse.java:12]: Comments should be written in English.

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogService.java [hunk 12]
```diff
@@ -0,0 +1,105 @@
...
+ * Copyright 2022 Holoinsight Project Authors. Licensed under Apache-2.0.
+ */
+package io.holoinsight.server.home.biz.service.log;
+
+import java.util.List;
+import java.util.Map;
+
+/**
+ * @author zzhb101
+ * @time 2023-04-07 11:33 上午
```
Comment (masaimu) [inline server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogService.java:11]: Comments should be written in English.

**Review by masaimu — APPROVED** [review]
LGTM

File: server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogContextResponse.java (hunk 3 above)
Comment (zzhb101) [reply server/home/home-service/src/main/java/io/holoinsight/server/home/biz/service/log/LogContextResponse.java]: removed

**Pull request merged**